### PR TITLE
Create dnslookupcheck.py

### PR DESCRIPTION
**Pull request by jfletchersimpson**
new dnslookup check collector.

### CODE REVIEW

**Review by Grumblesmith — CHANGES_REQUESTED** [review]
The default_config and default_config_help need corrections.

File: src/collectors/dnslookupcheck/dnslookupcheck.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        })
+        return config
+
+    def collect(self):
+        """
+        Overrides the Collector.collect method
+        """
+
+        #check to see if the dns name returns an IP address
+        for x in self.config['dnsAddressList']:
```
Comment (TheConnMan) [inline src/collectors/dnslookupcheck/dnslookupcheck.py]: `x` is not a descriptive variable name. I recommend changing to something like `dnsAddress`.

**Review by Grumblesmith — APPROVED** [review]

**Comment by Grumblesmith** [comment]
@TheConnMan I've updated the variable names, default_config and default_config_help sections, and shortened several lines which overran the PEP 8 line length limit. Could you take another look?

**Review by TheConnMan — APPROVED** [review]

**Comment by TheConnMan** [comment]
I was already looking at it.

**Comment by Grumblesmith** [comment]
Thanks!

**Pull request merged**